docs/src/pages/style/typography/typography.md

```diff
@@ -44,6 +46,14 @@ Material-UI default typography configuration only relies on 300, 400 and 500 fon
 
 {{"demo": "pages/style/typography/Types.js"}}
 
+## Custom and Self-Hosted Font
```

Comment:
Should we move this section into https://next.material-ui.com/customization/themes/#typography. I also believe we could change the headline hierarchy, so the important content can be linked and is better discoverable. `### Typography` -> `## Typography` and so on.

Reply:
- https://next.material-ui.com/style/typography/ focuses on the `Typography` component.
- https://next.material-ui.com/customization/themes/#typography focuses on `theme.typography`.

Reply:
I agree, it would make more sense in the theme customization section. I'll move it there and update the headline hierarchy. I also think it would be a good idea to provide a link from /style/typography to /customization/themes#typography to improve discoverability of typography customization in addition to improving search.

Reply:
Updated. Let me know if this is what you had in mind or what changes are needed.